### PR TITLE
Disable bootloader pattern

### PR DESCRIPTION
**Pull request by schubi2**
and enable systemd-boot if available

### CODE REVIEW

File: package/skelcd-control-MicroOS.changes [hunk 1]
```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Wed May 24 15:55:44 UTC 2023 - Stefan Schubert <schubi@suse.com>
+
+- Removed bootloader from default patterns.
```
Comment (imobachgs) [inline package/skelcd-control-MicroOS.changes]: Do you have any reference? Bugzilla entry or something like that.

**Review by imobachgs — COMMENTED** [review]
I am just missing a reference.

File: package/skelcd-control-MicroOS.changes (hunk 1 above)
Comment (schubi2) [reply package/skelcd-control-MicroOS.changes]: Yes, I have added it.

File: package/skelcd-control-MicroOS.changes [hunk 2]
```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Wed May 24 15:55:44 UTC 2023 - Stefan Schubert <schubi@suse.com>
+
+- Removed bootloader from default patterns. (JIRA#PED-1906)
```
Comment (imobachgs) [inline package/skelcd-control-MicroOS.changes]:
```suggestion
- Removed bootloader from default patterns. (jsc#PED-1906)
```

**Review by imobachgs — APPROVED** [review]

**Pull request merged**